curriculum/challenges/english/16-the-odin-project/top-learn-data-types-and-conditionals/learn-data-types-and-conditionals-question-e.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 65e97260484dd50f720e6fea
+videoId: LGQuIIv2RVA
```

Comment:
The video showing for me is an HTML video.

<img width="1202" alt="Screenshot 2024-03-12 at 9 34 00 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/738d8d27-38a4-4db5-88f0-bc3afdd12837">